content/docs/sidebar.json

```diff
@@ -730,14 +730,8 @@
               "remove-a-model-or-its-details"
             ]
           },
-          {
-            "slug": "team-collaboration",
-            "children": ["change-team-plan-and-size"]
-          },
-          {
-            "slug": "account-and-billing",
```

Comment:
[F] account-and-billing has become account-management again. `billing-and-payment` has been deleted.